### DISCUSSION TITLE
textobj-quote

### DISCUSSION THREAD

**Discussion by comod** [JetBrains/ideavim#532] [Q&A]
Hi!

Is it possible to use this Plugin? https://github.com/preservim/vim-textobj-quote

**Comment by AlexPl292** [comment]
Hello, this plugin is not yet supported. I've created a ticket for that: https://youtrack.jetbrains.com/issue/VIM-2726/Support-textobj-quote-plugin

**Reply by comod** [reply]
Is there any workaround possible for now?

**Reply by comod** [reply]
I do the following for now:
```
map <leader>v <esc>vi'
map <leader>V <esc>va'
map <leader>b <esc>vi"
map <leader>B <esc>va"
```

But it would be great to have one for both quote types
